cli/test/utils/broker.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const { MessageBroker, MESSAGE_EVENTS } = require('../../lib/MessageBroker');
+
+module.exports = {
+  createFakeBroker
+};
+
+function createFakeBroker() {
```

Comment:
currently only used in publish. this redirects events to be pushed to an array instead of the runtime console logger.